operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -237,9 +241,19 @@ size=11G features='1 queue_if_no_path' hwhandler='1 alua' wp=ro
 
 ### Node Personalization (Worker node)
 
-Node personalization is the pre-requisite step of SBPS solution where we need to first setup/ configure worker nodes as iSCSI targets (servers)
+Node personalization is the pre-requisite step of SBPS solution where we need to first setup/configure worker nodes as iSCSI targets (servers)
 with necessary provisioning, configuration and enable required components. The required RPMs for `targetcli` command / LIO are part of NCN node image in CSM 1.6 and SBPS Marshal agent gets installed during node personalization via CFS Ansible play books.
 
+This can be done in two ways:
+
+a) Auto setup with bootprep:
+
+By default worker node personalization of iSCSI SBPS is done during CSM install/upgrade (using IUF). It is initiated during bootprep
+(management-nodes-rollout) in order to do worker node personalization automatically during boot time.
+
+b) Manual setup with CFS config/session:
+
+Worker node personalization can be done post CSM install with CFS config/session.
 Please refer [Node Personalisation](https://github.com/Cray-HPE/docs-csm/blob/release/1.6/operations/configuration_management/iSCSI_SBPS_Node_Personalization.md) for the details.
```

Comment:
```suggestion
Please refer [Node Personalization](https://github.com/Cray-HPE/docs-csm/blob/release/1.6/operations/configuration_management/iSCSI_SBPS_Node_Personalization.md) for the details.
```

Reply:
Also, it should be "Please refer **_to_** [Node"

Reply:
Done